src/Microsoft.DotNet.Build.Tasks/PackageFiles/Symbols.targets

```diff
@@ -403,4 +448,24 @@
                        Condition="'@(CatalogFile)'!=''" />
   </Target>
 
+  <!--
+    Generates Windows PDBs from Portable PDBs.
+    
+    [In]
+    @(PortableFileToConvert)
+      * ItemSpec: The DLL file associated with a Portable PDB to convert.
+      * PdbPath: The path to the Portable PDB file to convert.
+      * TargetPath: The output path for the generated Windows PDB. Full filename.
+    @(ConversionOptions) [optional]
+      * ItemSpec: An entry in the Microsoft.DiaSymReader.Tools.PdbConversionOptions flags enum to
+        use for all conversions performed.
+  -->
+  <Target Name="CreateWindowsPdbsFromPortablePdbs">
+    <Error Text="BuildTools does not support Portable PDB conversion to Windows PDB in .NET Core. Run msbuild using the desktop framework."
```

Comment:
It's `core` when running on .NET Core, even on Windows. Running the task fails to do some interop stuff because the DLLs it needs aren't present.

Here's the error:

```
The "ConvertPdbsToPortablePdbs" task failed unexpectedly.\r [C:\git\corefx\build.proj]
System.InvalidCastException: Unable to cast COM object of type 'System.__ComObject' to interface type 'Microsoft.DiaSymReader.ISymUnmanagedWriter8'. This operation failed because the QueryInterface call on the COM component for the interface with IID '{5BA52F3B-6BF8-40FC-B476-D39C529B331E}' failed due to the following error: No such interface supported (Exception from HRESULT: 0x80004002 (E_NOINTERFACE)).\r [C:\git\corefx\build.proj]
at Microsoft.DiaSymReader.SymReaderFactory.CreateWindowsPdbWriter(Object pdbStream, Object metadataProvider) in E:\A\_work\7\s\src\Microsoft.DiaSymReader.Converter\SymReader\SymReaderFactory.cs:line 89\r [C:\git\corefx\build.proj]
...
```

I'm not really a fan of a warning or silent continue: I feel like I'd rather make sure people confirm for certain that they don't want to produce Windows PDBs. A concern would be someone trying to change symbol publishing to run on Core without realizing that this is one of the effects.

I'll add a property for skipping this process entirely, though. I suppose you could override the target, but a property would be clearer.